src/routes/auth/entities/schemas/verify-auth-message.dto.ts

```diff
@@ -0,0 +1,4 @@
+import { VerifyAuthMessageDtoSchema } from '@/routes/auth/entities/schemas/verify-auth-message.dto.schema';
+import { z } from 'zod';
+
+export type VerifyAuthMessageDto = z.infer<typeof VerifyAuthMessageDtoSchema>;
```

Comment:
nitpick: to merge this file with the entity one (less reference jumping around the project).

Reply:
Merged in 719b0f301fa3901b5ec468b7c90c04936b592f8c.